### ISSUE TITLE
page 475, R code 14.45

### ISSUE THREAD

**Issue by troelsring** [rmcelreath/rethinking#245]
I believe m14.8 does not have parameter bp but rather b as also shown in R code 14.40 although the text just below that still speaks of bp. The result is that code 14.45 does not find lambda. Annoyingly, taking post$b instead of post$bp doesn't seem to be right - at least figure 14.12 right is not reproduced. It is tough for an old clinician like me but the account is so interesting, thanks a lot 
Troels

**Comment by jnmaloof** [comment]
I think that to reproduce the figure the code in 14.45 should be:

```
lambda <- sapply( logpop.seq , function(lp) post$a*exp(lp)^post$b/post$g )
```

**Comment by troelsring** [comment]
Thanks a lot - this improves the figure but as I can see it still doesn't reproduce figure 14.12 right - e.g. the upper confidence is placed in the middle between Yap and Chuuk, but with the suggested solution much closer to Chuuk? Is this just random variation?

**Comment by jnmaloof** [comment]
When I run it the upper line is just above Chuuk

**Comment by troelsring** [comment]
Indeed, and that is different from the figure in the book with the upper line just as far from Chuuk as from Yap?

**Comment by jnmaloof** [comment]
Oh, sorry, I read your email backwards and didn't look at the book until now.  Hopefully @rmcelreath will be able to help.

**Comment by nnawata** [comment]
How about ```lambda = sapply ( logpop.seq, function(lp) exp(post$a + post$b*lp))``` ?
Looks OK to me.
